app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if person.get("wife"):
+            wife = Person.people[person.get("wife")]
+            husband = Person.people[person.get("name")]
+            husband.wife = wife
+            wife.husband = husband
+    return list(Person.people.values())
```

Comment:
btw, you can create a variable in the `$11` line:
```
people_list = [Person(person["name"], person["age"]) for person in people]
```

and here `return people_list`. WDYT?